client/ts/src/framer/iterator.ts

```diff
@@ -105,6 +107,7 @@ export class Iterator {
       command: Command.Open,
       keys: adapter.keys,
       bounds: new TimeRange(tr),
+      chunkSize: chunkSize || 5e5,
```

Comment:
Use as a default argument in the function instead. Just like python. Also, when doing an op like this (if you do do it), you should use nullish coalescing instead (`chunkSize ?? 5e5`).  As with the note on python, let's make the auto chunk size significantly smaller.